libs/application-generic/src/services/analytics.service.ts

```diff
@@ -71,6 +71,22 @@ export class AnalyticsService {
     });
   }
 
+  updateGroup(
+    userId: string,
+    groupId: string,
+    traits: Record<string, string | string[]>,
+  ) {
+    if (!this.segmentEnabled) {
+      return;
+    }
+
+    this.segment.group({
```

Comment:
wont we be missing group of : id, name, createdAt, organizationType, companySize?

Reply:
It's only updating new fields here (acts as patch in a way) so not really a problem